examples/oneapi/cpp/source/pca/pca_dense_batch.cpp

```diff
@@ -21,22 +21,35 @@
 
 using namespace oneapi;
 
-int main(int argc, char const *argv[]) {
-    const std::string data_file_name = get_data_path("pca_normalized.csv");
-
-    const auto data = dal::read<dal::table>(dal::csv::data_source{data_file_name});
-
-    const auto pca_desc = dal::pca::descriptor<>()
+template <typename Method>
+void run_compute(dal::table x_train, const char *method_name) {
+    const auto pca_desc = dal::pca::descriptor<float, Method>()
         .set_component_count(5)
         .set_deterministic(true);
 
-    const auto result = dal::train(pca_desc, data);
+    const auto result_train = dal::train(pca_desc, x_train);
+
+    std::cout << method_name << '\n' << std::endl;
```

Comment:
If you use `std::endl`, use it consistently
```suggestion
    std::cout << method_name << std::endl << std::endl;
```